metis-normalization/src/main/java/eu/europeana/normalization/dates/DateNormalizationResult.java

```diff
@@ -69,6 +69,7 @@ public String toString() {
         + edtfDate + "]";
   }
 
+
   // TODO: 22/07/2022 Double check if this is correct, it has not been tested since it's used in the untested generic property method
```

Comment:
That's the method for generic property.
A range of only years(without a month and day), it should be considered valid.

From documentation:
`A normalised date is only accepted if it is considered reliable by verifying the following conditions (this verification is implemented in the method isCompleteDate() of the Match class):
The date contains the year, month and day.
If it is a range, the start and end dates both contain the year, month and day, or both contain only the year. For example, ‘1980-05-06/1980-06-20’ or ‘1980/1981’.`